app/views/components/radios.scala.html

```diff
@@ -31,18 +31,9 @@
 )(implicit messages: Messages, userRequest: UserRequest[_], appConfig: AppConfig)
 
 @legend = {
-  <h1 class="govuk-fieldset__heading">
-    <span class="govuk-caption-l penalty-information-caption" id="penalty-information">
-      @{
-        messages("penaltyInformation.headerText",
-          PenaltyTypeHelper.getKeysFromSession().get.head,
-          PenaltyTypeHelper.getKeysFromSession().get(1),
-          PenaltyTypeHelper.getKeysFromSession().get.last
-        )
-      }
-    </span>
-    <span class="govuk-heading-l govuk-!-margin-bottom-0" id="page-heading">@legendContent</span>
-  </h1>
+    <h1 class="govuk-fieldset__heading" id ="page-heading">
+        @legendContent
+    </h1>
 }
 
 @govukRadios(Radios(
```

Comment:
Could you rename showPenaltyInformation to be something like legendAsPageHeading?